icu4c/source/data/buildtool/filtration.py

```diff
@@ -281,8 +281,10 @@ def apply_to_requests(self, all_requests):
                 continue
             if request.tool != IcuTool("genrb"):
                 continue
+            if not request.input_files:
+                continue
             self._set_files(request.input_files)
-            request.dep_targets += [self.filter_files]
+            request.dep_targets += [self.filter_files[:]]
```

Comment:
FYI: I had a hunch for what `[:]` does but had to look it up. Using `.copy()` would be clearer.